### DISCUSSION TITLE
Supported version of .NET

### DISCUSSION THREAD

**Discussion by jeremyhagan** [microsoft/vscode-azurearmtools#1803] [General]
Since .NET core 3.1 is EOS, what is the recommended version to install? We are in an environment where the .NET installer can't download stuff like that.

**Comment by StephenWeatherford** [comment]
Not sure what you're asking - are you asking about the vscode ARM Extension's version?  The early December release will use dotnet 8.  Where are you seeing 3.1?

**Reply by jeremyhagan** [reply]
in the wiki
https://github.com/microsoft/vscode-azurearmtools/wiki/Troubleshooting-DotNet-Acquisition

**Reply by jeremyhagan** [reply]
I noticed that the .NET installer used 7.0.20, so I have run with that for now.

**Reply by StephenWeatherford** [reply]
Thanks, fixed the wiki (anything higher than 7, or soon 8, should work).  But you *shouldn't* have to install .net yourself unless you're trying to work around an issue.  What are you seeing?

**Reply by jeremyhagan** [reply]
No internet access to download it from that environment

**Reply by StephenWeatherford** [reply]
Can I close the discussion?  Thx

**Comment by jeremyhagan** [comment]
Thanks for your help
